tests/integration/test_dependencies_compatibility_protobuf.py

```diff
@@ -457,3 +593,79 @@ async def test_protobuf_references_latest(registry_async_client: Client) -> None
     )
     assert res.status_code == 200
     assert "id" in res.json()
+
+
+async def test_protobuf_customer_update_references(registry_async_client: Client) -> None:
```

Comment:
`test_protobuf_customer_update_when_having_references` or similar. The test does not update the references, but tests a update case with references.